shifthelper/tools/__init__.py

```diff
@@ -76,7 +76,23 @@ def fetch_users_awake():
         wrap_exception=True
     )
     def retry_fetch_fail_after_30sec():
-        return requests.get('https://ihp-pc41.ethz.ch/iAmAwake').json()
+        return requests.get('http://webinterface/iAmAwake').json()
```

Comment:
Maybe we should configure this url in the config (keeping local testing in mind).